docs/user-guide/helm.md

```diff
@@ -26,6 +26,33 @@ in the form of `-p PARAM=VALUE`. For example:
 argocd app set helm-guestbook -p service.type=LoadBalancer
 ```
 
+## Helm Release Name
+
+By default the Helm release name is equal to the Application name to which it belongs. Sometimes, especially on a centralised ArgoCD, 
+you may want to override that  name, and it is possible with the `release-name` flag on the cli:
+
+```bash
+argocd app set helm-guestbook --release-name myRelease
+```
+
+ or using the releaseName for yaml:
+
+```yaml
+source:
+    helm:
+      releaseName: myRelease
+```
+
+```diff
+- Important notice on overriding the release name
```

Comment:
minor - you can use a `!!! warning` admonition block here, see https://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
I am not sure admonition extension works on Github, I just tried on a separate repo to display an warning and it didn't work. Maybe I am missing something?